packages/internal/src/utils.js

```diff
@@ -266,10 +266,10 @@ harden(bindAllMethods);
  * A more constrained version of {deeplyFulfilled} for type safety until https://github.com/endojs/endo/issues/1257
  * Useful in starting contracts that need all terms to be fulfilled in order to be durable.
  *
- * @type {<T extends {}>(unfulfilledTerms: T) => import('@endo/eventual-send').ERef<DeeplyAwaited<T>>}
+ * @type {<T extends {}>(unfulfilledTerms: T) => Promise<DeeplyAwaited<T>>}
  */
 export const deeplyFulfilledObject = obj => {
```

Comment:
An open question from the private chat is whether `deeplyFulfilledObject` should also be made async. Currently this PR does not. Reviewers, please let me know if you think this PR should make that change.

Reply:
I think it makes sense to change `deeplyFulfilledObject` too.

Reply:
Done